rules/invite-collaborator.ts

```diff
@@ -6,7 +6,7 @@ Holy buckets, @${username} — we just merged your PR to Gatsby! 💪💜
 Gatsby is built by awesome people like you. Let us say “thanks” in two ways:
 
  1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (Currently we’ve got a couple t-shirts available, plus some socks that are really razzing our berries right now.)
- 2.  **We just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.
+ 2.  **We just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. Accept the invite by visiting https://github.com/orgs/gatsbyjs/invitation . By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.
```

Comment:
Why not make this an actual link? i.e.

```md
Accept the invite by [visiting this link](https://github.com/orgs/gatsbyjs/invitation)
```

Reply:
@DSchau I kind of like the idea of keeping it a link to make it obvious where you're headed when you click it. We can revisit later if we decide that's not the best approach, though.